recipes/graylog-illuminate/files/post-install

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -e
+
+# Flag to indicate if this is an upgrade
+upgrade="false"
+
+case "$1" in
+	# DEB based systems
+	configure)
+		if [ -z "$2" ]; then
+			upgrade="false"
+		else
+			upgrade="true"
+		fi
+		;;
+	abort-deconfigure|abort-upgrade|abort-remove)
+		;;
+	# RPM based systems
+	1)
+		# Installation
+		upgrade="false"
+		;;
+	2)
+		# Upgrade
+		upgrade="true"
+		;;
+	*)
+		echo "[ERROR] post-install script called with unknown argument: '$1'"
+		exit 1
+		;;
+esac
+
+chmod 0644 /etc/graylog/graylog_illuminate_core/*
```

Comment:
Are these standard directories we use in illuminate? If not, I would prefer something like the following. The current naming looks redundant.

- `/etc/graylog/illuminate/core`
- `/etc/graylog/illuminate/windows`
- ...

Reply:
Yes, those are the directories the tarball creates. 

@miwent, can these directories be changed at all? I am not familiar with the process that picks these files up.

Reply:
We can change these in a future release, but for now they are tied with the lookup configurations in the content packs. I've created an issue for this: [link](https://github.com/Graylog2/graylog-project-illuminate/issues/160)